### DISCUSSION TITLE
Spectrum RGB - Edit existing effects

### DISCUSSION THREAD

**Discussion by urbanwks** [BartoszCichecki/LenovoLegionToolkit#963] [General]
Apologies if I'm completely missing it, but is it possible to add/remove keys from an existing effects group? Or do you have to delete the existing effect and re-add it with the change you want? If it's not currently an option I'd like to add a feature request for it. This program is exactly what i need, particularly the export/import of custom keyboards (mine keep resetting for some reason). Thanks for the amazing work!

**Comment by BartoszCichecki** [comment]
> Or do you have to delete the existing effect and re-add it with the change you want?

Yes, that is the cause, due to quite complex format that is used internally. That is also why I do not plan on "editing" keys that are part of an effect.
